src/annotator/test/guest-test.js

```diff
@@ -754,18 +804,18 @@ describe('Guest', () => {
   });
 
   describe('#createAnnotation', () => {
-    it('creates an annotation if host calls "createAnnotationIn" RPC method', () => {
-      const guest = createGuest();
-      sinon.stub(guest, 'createAnnotation');
```

Comment:
Remove this reliance on mocking a public method. Instead a check that the 'createAnnotation' RPC event is sent to the sidebar.